packages/dart/npt_flutter/pubspec.yaml

```diff
@@ -16,7 +16,16 @@ publish_to: "none" # Remove this line if you wish to publish to pub.dev
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
 # In Windows, build-name is used as the major, minor, and patch parts
 # of the product and file versions while build-number is used as the build suffix.
-version: 1.1.3+0
+version: 1.1.4+0
```

Comment:
version number should be 1.1.4+4. The +4 indicate it is the 4th build. validation will fail if that number is not greater than the previous number.

Reply:
Oh... you're right...

![CleanShot 2024-12-06 at 20 41 10@2x](https://github.com/user-attachments/assets/6421996e-6eb6-4c90-ab5e-da512625f419)

Macos is just weird, iOS' way is the normal way things tend to be done... a build number must only be unique to each version number. But for some reason, with macos... build number must be unique, regardless of the version number...